tests/detectors/huggingface/test_method_process_token_classification.py

```
@@ -0,0 +1,188 @@
+        """Both env and request safe_labels should be merged via union."""
+        model_dir = os.path.join(
+            os.environ["MODEL_DIR"], "bert/BertForTokenClassification"
+        )
+        os.environ["MODEL_DIR"] = model_dir
+        os.environ["SAFE_LABELS"] = '["O"]'
+        try:
+            detector = Detector()
+            text = "This is a test."
+            all_labels = list(detector.model.config.id2label.values())
+            # Request safe_labels covers all labels except "O" (already in env)
+            request_safe = [l for l in all_labels if l != "O"]
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)